integration/monai/monai_nvflare/nvflare_stats_handler.py

```diff
@@ -40,11 +40,11 @@
 DEFAULT_TAG = "Loss"
 
 
-class NVFlareStatsHandler(LogWriterForMetricsExchanger):
+class NVFlareStatsHandler(LogWriter):
```

Comment:
changes in this PR LGTM as a first stage.

We can think of these questions:
- Is `NVFlareStatsHandler` a `LogWriter`? should we inherit it? or should we use some composition?